mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +26,41 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should image open?
```

Comment:
Delete "?" at the end

Add default value

Reply:
Ok